build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -0,0 +1,184 @@
+                     --output $(SdkOutputDirectory)
+                     --configuration $(Configuration)
+                     --version-suffix $(CommitCount)
+                     $(SrcDirectory)/dotnet" />
+
+      <!-- Corehostify Binaries -->
+      <FixModeFlags Dir="$(SdkOutputDirectory)" />
+
+      <Exec Command="%(Stage.DotnetExe) publish
+                     --output $(SdkOutputDirectory)
+                     --framework netcoreapp1.0
+                     $(SrcDirectory)/compilers" />
+
+      <Copy SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
+            DestinationFiles="$(SdkOutputDirectory)/csc.dll" />
+
+      <Move SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
+            DestinationFiles="$(SdkOutputDirectory)/csc.exe" />
+
+      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.deps.json"
+            DestinationFiles="$(SdkOutputDirectory)/csc.deps.json" />
+
+      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.runtimeconfig.json"
+            DestinationFiles="$(SdkOutputDirectory)/csc.runtimeconfig.json" />
+
+      <ChangeEntryPointLibraryName 
+            DepsFile="$(SdkOutputDirectory)/csc.deps.json"
+            NewName="csc.deps.json" />
+
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/csc.deps.json"
+                                   SectionName="runtimeTargets"
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/dotnet.deps.json"
+                                   SectionName="runtimeTargets"
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
+
+      <!-- cleanup compilers project output we don't need -->
+      <CleanPublishOutput Path="$(SdkOutputDirectory)"
+                          Name="compilers"
+                          DeleteRuntimeConfigJson="true"
+                          DeleteDepsJson="true" />
+
+      <!-- Copy Host to SDK Directory -->
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(DotnetHostBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/corehost$(ExeSuffix)" />
+
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(DotnetHostFxrBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/$(DotnetHostFxrBaseName)" />
+
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(HostPolicyBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/$(HostPolicyBaseName)" />
+
+      <CrossgenDirectory CoreCLRVersion="$(CoreCLRVersion)"
+                         JitVersion="$(JitVersion)"
+                         SharedFrameworkNameVersionPath="$(SharedFrameworkNameVersionPath)"
+                         SdkOutputDirectory="$(SdkOutputDirectory)" />
+
+      <!-- Generate .version file -->
+      <WriteLinesToFile File="$(SdkOutputDirectory)/.version"
+                        Lines="$(CommitHash);$(SdkVersion)"
```

Comment:
Do we still want a trailing newline?  It was in there before, could possibly break someone consuming this file.